src/components/AdditionalMaterials/index.tsx

```diff
@@ -0,0 +1,39 @@
+import React from 'react';
+import { HelpCircle } from 'react-feather';
+import { Styles } from '../../constants';
+import { SbNavLink, SbNavlinkProps } from '../SbNavLink';
+
+export interface AdditionalMaterialsProps {
+  links: SbNavlinkProps[];
+}
+
+export const AdditionalMaterials: React.SFC = (): JSX.Element => {
+  const sampleItemsContent = 'Sample Items, Scoring Guides and Rubrics';
+  const annotatedContent = 'Annotated Student Work';
+  const digitalLibraryContent = 'Digital Library Resources';
```

Comment:
Just to make this a bit cleaner, if the string s aren't going to change. Take a page from Evan's book on the api side and declare the strings outside of the component in an enum. 
```
export enum Content {
  SampleItems = 'Sample Items, Scoring Guides and Rubrics',
  Annotated = 'Annotated Student Work',
  DigitalLibrary = 'Digital Library Resources'
}

<SbNavLink url={''} content={buildContent(Content.SampleItems)} />
```

Reply:
We probably wont need to export it though.

Reply:
This has been updated